lld/ELF/Relocations.cpp

```diff
@@ -804,6 +804,13 @@ static bool maybeReportUndefined(Ctx &ctx, Undefined &sym,
   if (ctx.arg.unresolvedSymbols == UnresolvedPolicy::Ignore && canBeExternal)
     return false;
 
+  // Skip undefined symbols from list
+  for (const auto &ignoredUndef : ctx.arg.unresolvedSymbolsList) {
+    if (ignoredUndef == sym.getName()) {
```

Comment:
omit braces in this case. 

ctx.arg.unresolvedSymbolsList can be a DenseSet<CachedStringRef> to improve performance when the list is long.
I'd file a binutils feature request before rushing to add an option